qwik-graphql-tailwind/src/routes/auth/redirect/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { component$, useClientEffect$ } from '@builder.io/qwik';
+import type { DocumentHead } from '@builder.io/qwik-city';
+import { AUTH_TOKEN, GET_TOKEN_URL } from '~/utils/constants';
+import * as styles from '../signin/signin.classNames';
+
+export default component$(() => {
+  useClientEffect$(async () => {
```

Comment:
I know I'm the one who suggested `useClientEffect`, but I wonder whether [useMount$](https://qwik.builder.io/tutorial/hooks/use-mount/) may be better here as hypothetically `useClientEffect` could fire multiple times.

Reply:
Tho i didn't not notice any multiple calls, i just tested with useMount and it threw an error `window is undefined`